wordpress-mobile/WordPress-iOS
# Associate a custom domain to a site with a business plan

Ref: #9630 

Many of business sites don't have a custom domain associated even though it's free with their plan. We want to prompt and allow users that have already bought a business plan to associate a custom domain to their site.

**Prerequisite:** 

Have a site with the below criteria:
- It is a WordPress.com site
- It has a business plan
- It is not private
- The user has admin rights for the site

**To test:** 
- Open My Sites
- Tap on the site having above criteria
- See that CONFIGURE section has Plugins option between People and Settings.
- Tap on Plugins and see that Plugins page opens
- Tap on one of the Plugins
- Tap on Install
- Confirm that a Popup is displayed with a Register domain option
- Select the Register domain option and start the flow
- At the end of the flow it should return to plugin page
